src/rulesets/oas/functions/oasPathParam.ts

```diff
@@ -103,6 +103,7 @@ export const oasPathParam: IFunction = (targetVal, _options, paths, vals) => {
 
   // keep track of normalized paths for verifying paths are unique
   const uniquePaths: object = {};
+  const validOperationKeys = ['get', 'head', 'post', 'put', 'patch', 'delete', 'options', 'trace'];
```

Comment:
@bzmw Could you please also update the `path-params` section of `openapi-rules.md` doco to quickly explain that very non standard HTTP verb will be silently ignored?

Reply:
Done!